### PR TITLE
Azure Auth - rotate-root documentation

### PR DESCRIPTION
**Pull request by Zlaticanin**

### CODE REVIEW

File: website/content/api-docs/auth/azure.mdx [hunk 1]
```diff
@@ -111,6 +111,28 @@ $ curl \
...
+
+There are no parameters to this operation.
+
+### Sample Request
+
+```shell-session
+$ curl \
+  --header "X-Vault-Token: ..." \
+  --request POST \
+  http://127.0.0.1:8200/v1/azure/rotate-root
```
Comment (fairclothjm) [inline website/content/api-docs/auth/azure.mdx:133]: I would suggest
```suggestion
  https://127.0.0.1:8200/v1/auth/azure/rotate-root
```
to keep it similar to the other sample requests on this page

**Comment by austingebauer** [comment]
We'll want to document permissions that the root credential needs similar to how that's done in [Azure secrets](https://developer.hashicorp.com/vault/docs/secrets/azure#ms-graph-api-permissions). 

Azure auth only needs the following for rotate root:

| Permission Name               | Type        |
| ----------------------------- | ----------- |
| Application.ReadWrite.All     | Application |

**Review by vinay-gopalan — APPROVED** [review]
LGTM

File: website/content/docs/auth/azure.mdx [hunk 2]
```diff
@@ -56,6 +56,15 @@ or [`resource_id`](/vault/api-docs/auth/azure#resource_id) parameters are used o
 
 ## Authentication
 
+### Root Credentials API Permissions
```
Comment (austingebauer) [inline website/content/docs/auth/azure.mdx:59]: I'm realizing now that the "Authentication" section here has to do with actually authenticating (i.e., logging in) to via the plugin. I think this recommendation around API permissions is out of place here.

I suggest to place this in the above section with "Prerequisites". Perhaps we can add a "###" level section called "Role Assignments" on line 41 and a "###" level section called "API Permissions" on line 56 that has the content you've added below here. Let me know if that doesn't make sense 🙂

Comment (Zlaticanin) [reply website/content/docs/auth/azure.mdx]: That makes sense. I just moved it under the Prerequisites and added new sections. Let me know if this is not what you had in mind 😃 
Thank you!

**Review by austingebauer — APPROVED** [review]
Thanks, @MilenaHC 👍

**Pull request merged**